libs/api-interfaces/src/lib/aggregate-views/view-models/user-management/coscrad-contributor/coscrad-contributor.view-model.interface.ts

```diff
@@ -0,0 +1,13 @@
+import { IBaseViewModel } from '../../base.view-model.interface';
+
+export interface ICoscradContributorViewModel extends IBaseViewModel {
+    label: string;
+
+    description: string;
+
+    fullName: string;
```

Comment:
@NonEmptyString(...)

Reply:
Oh wait- this is the interface. You would only decorate the fields in the implementation class.